samples/crypto/ecjpake/src/main.c

```diff
@@ -122,30 +122,36 @@ psa_status_t pake_setup(psa_pake_operation_t *op, psa_pake_cipher_suite_t *cs, c
 		return status;
 	}
 
-	status = psa_pake_set_password_key(op, *password);
-	if (status != PSA_SUCCESS) {
-		LOG_INF("psa_pake_set_password_key failed. (Error: %d)", status);
-		return status;
-	}
-
 	return PSA_SUCCESS;
 }
 
 psa_status_t do_key_derivation(psa_pake_operation_t *op, uint8_t *key_buffer,
 			       size_t key_buffer_size)
 {
+	psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+	mbedtls_svc_key_id_t key;
```

Comment:
This should really be `psa_key_id_t`. I think you had some issues with that? If that truly does not work. can you add a comment about why this is not according to psa spec? (psa_key_id_t is used other places in this file)

Reply:
Yeah got some warning during the change but was due to my setup, changed it now to `psa_key_id_t`